CatoNetwork/manifest.json

```
@@ -7,7 +7,7 @@
                 "type": "string"
             },
             "account_id": {
-                "description": "Account Id to work with Cato API",
+                "description": "Account Id (4 digits) to work with Cato API",
```

I suggest adding a pattern to validate the account_id.
```suggestion
                "description": "Account Id (4 digits) to work with Cato API",
                "pattern": "^[0-9]{4}$"
```